### ISSUE TITLE
About Camera

### ISSUE THREAD

**Issue by hcl013** [hku-mars/ImMesh#5]
Hello
Can I ask you the model of the camera you are using, in the paper?

**Comment by ziv-lin** [comment]
Yes, as mentioned by @FPSychotic, In our Experitment-1, we use the camera Flir blackfly the same as of R3LIVE for providing the camera preview, while in our Application-2, we mount hikvision on our drone platform.
